src/user_defines.txt

```diff
@@ -9,8 +9,9 @@
  
 
 ### BINDING PHRASE: ### 
-
--DMY_BINDING_PHRASE="default ExpressLRS binding phrase"
+# Uncomment the line below to use a hardcoded binding phrase
+# Leave commented to use traditional binding
```

Comment:
I'd suggest calling it "automated" binding or something like that. Traditional conflicts with "old way of binding".

Reply:
Nah. "Automated" doesn't describe it well. It basically is traditional (old way) of binding, except we use a slightly different trigger on the rx for entering bind mode.

Reply:
"Simple Bind"